tests/plugins/widget/manual/keystroke.md

```diff
@@ -0,0 +1,37 @@
+@bender-tags: widget, bug, 4.14.1, 3998
+@bender-ui: collapsed
+@bender-ckeditor-plugins: toolbar, wysiwygarea, sourcearea
```

Comment:
This test passes event without fixes you have applied - this is caused by the fact that the fixes are in widget plugin code which is not included here :see\_no\_evil: Always make sure that the test fully covers the issue (so fails without the fix).